scripts/clean_e2e.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
```

Comment:
I'm feeling like we made a mistake not investigating the bazel integration test scheme from Ittai more. We've built a build system for the e2e/examples tests now

Reply:
I think this is better than what we had and it buys us some time. The CI time is down with this setup and testing & developing locally is much better. The clean scripts are useful for local development only.